invenio_vocabularies/config.py

```diff
@@ -74,6 +74,33 @@
 }
 """Awards allowed identifier schemes."""
 
+VOCABULARIES_AWARD_OPENAIRE_FUNDERS = {
```

Comment:
comment:

at some point we will have an awards dump in yaml, made based on some filtering and we will load it to Zenodo. Why don't we add this as a property e.g. "openaire_id", which can be resolved from DB? This would mean that new funders/updates just need to be processed as any other vocabulary update.


p.s. sorry for "eavesdropping" 😇